example-get-started/code/.github/workflows/cml.yaml

```
@@ -14,13 +14,18 @@ jobs:
           echo "# CML Report" > report.md
 
           echo "## Plots" >> report.md
-          dvc plots diff bigrams-experiment workspace \
+          dvc plots diff main \
             --show-vega --targets prc.json > vega.json
           vl2svg vega.json prc.svg
           cml publish prc.svg --title "Precision & Recall" --md >> report.md
 
           echo "## Metrics" >> report.md
-          echo "### bigrams-experiment → workspace" >> report.md
-          dvc metrics diff bigrams-experiment --show-md >> report.md
+          if [[ "${GITHUB_REF}" == "refs/heads/main" ]] ; then
+            PREVIOUS_REF='HEAD^1'
```

I tend to prefer `~` because it work on Windows & Linux (`^` is an escape char on Win)

```suggestion
            PREVIOUS_REF='HEAD~1'
```